ingest/usecase/ingest_usecase.go

```diff
@@ -452,6 +452,11 @@ func processSQSModelMut(sqsModel *sqsdomain.SQSPool) error {
 		alloyedDenom := cosmWasmModel.Data.AlloyTransmuter.AlloyedDenom
 
 		sqsModel.PoolDenoms = append(sqsModel.PoolDenoms, alloyedDenom)
+
+		// Process the alloyed pool
+		if err := processAlloyedPool(sqsModel); err != nil {
+			return err
+		}
```

Comment:
Note: this should pre-compute some of the normalization scaling factors needed for weight construction

Reply:
I guess you mean that we move this to ingest time?